oggsound/include/OgreOggSoundManager.h

```diff
@@ -722,19 +733,26 @@ namespace OgreOggSound
 			while(!mShuttingDown)
 			{	
 				{
+/*
 #ifdef POCO_THREAD
 					Poco::Mutex::ScopedLock l(OgreOggSoundManager::getSingletonPtr()->mMutex);
 #else
 					boost::recursive_mutex::scoped_lock lock(OgreOggSoundManager::getSingletonPtr()->mMutex);
 #endif
+*/
+					OGRE_LOCK_MUTEX_NAMED(OgreOggSoundManager::getSingletonPtr()->mMutex, lock);
```

Comment:
these also need to be
```suggestion
					OGRE_WQ_LOCK_MUTEX_NAMED(OgreOggSoundManager::getSingletonPtr()->mMutex, lock);
```